daffodil-core/src/main/scala/org/apache/daffodil/runtime1/ChoiceTermRuntime1Mixin.scala

```diff
@@ -136,7 +136,7 @@ trait ChoiceTermRuntime1Mixin { self: ChoiceTermBase =>
               // because if so, we have a true ambiguity here.
               case sg: SequenceTermBase => {
                 val nonOVCEltChildren = sg.groupMembers.filter {
-                  case erd: ElementRuntimeData => erd.outputValueCalcExpr.isEmpty
+                  case eb: ElementBase => !eb.isOutputValueCalc
```

Comment:
Suggest add type to filter test function: 
``` 
sg.groupMembers.filter{ t: Term => t match {
.... cases....
   }
}
```
extra braces, but worth it for the type checking.